metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/MarcXmlHandler.java

```diff
@@ -32,7 +32,7 @@
  * @author Markus Michael Geipel
  *
  */
-@Description("A marc xml reader")
+@Description("A MARC XML reader. To read marc data without namespace specification set option `namespace=\"\"`")
```

Comment:
Does setting `namespace=""` actually result in a `null` namespace? Or an empty string?

Reply:
You are right, the `namespace` can only be set as empty via `flux`, not `null`.
I've updated the commit message and make that more explicit in the javadocs.